apps/examples/testcase/le_tc/network/itc_net_connect.c

```diff
@@ -304,40 +315,52 @@ static void itc_net_connect_p_multiple_clients(void)
 	int id2 = 2;
 	int id3 = 3;
 	int ret;
-	int *ptr_num_clients = &num_clients;
-	int *ptr_id1 = &id1;
-	int *ptr_id2 = &id2;
-	int *ptr_id3 = &id3;
 	int *pret = OK;
 
+	_initialize_test();
```

Comment:
Is it okay not to check the initialization?
In other tc, you checked and return if failed.

Reply:
Thanks I'll modify it